soap/src/java/com/zimbra/soap/admin/type/VolumeExternalOpenIOInfo.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2022 Synacor, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software Foundation,
+ * version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ * ***** END LICENSE BLOCK *****
+ */
+
+package com.zimbra.soap.admin.type;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+
+import com.zimbra.common.soap.AdminConstants;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+@XmlAccessorType(XmlAccessType.NONE)
+
+public class VolumeExternalOpenIOInfo extends BaseExternalVolume {
+
+    /**
+     * @zm-api-field-description Specifies the standard HTTP URL for OpenIO
+     */
+    @XmlAttribute(name=AdminConstants.A_VOLUME_URL /* url */, required=false)
+    private String url;
+
+    /**
+     * @zm-api-field-description Specifies OpenIO account name
+     */
+    @XmlAttribute(name=AdminConstants.A_VOLUME_ACCOUNT/* account */, required=false)
+    private String account;
+
+    /**
+     * @zm-api-field-description Specifies OpenIO namespace
+     */
+    @XmlAttribute(name=AdminConstants.A_VOLUME_NAMESPACE /* namespace */, required=false)
+    private String namespace;
```

Comment:
namespace variable should be nameSpace, accordingly setters/getters will get affected..

Reply:
done